packages/docs/docs/careplans/tasks/tasks.md

```diff
@@ -0,0 +1,140 @@
+| Registered Nurse           | 224535009 ([Registered Nurse](https://browser.ihtsdotools.org/?perspective=full&conceptId1=224535009&edition=MAIN/2023-03-31&release=&languages=en))           | 29-1141 (Registered Nurses)                               |
+| Care Coordinator           | 768820003 ([Care Coordinator](https://browser.ihtsdotools.org/?perspective=full&conceptId1=768820003&edition=MAIN/2023-03-31&release=&languages=en))           | 11-9111 (Medical and Health Services Managers)            |
+| Care Team Coordinator      | 768821004 ([Care Team Coordinator](https://browser.ihtsdotools.org/?perspective=full&conceptId1=768821004&edition=MAIN/2023-03-31&release=&languages=en))      | 11-9111 (Medical and Health Services Managers)            |
+| Medical Billing Specialist | 1251542004 ([Medical Coder](https://browser.ihtsdotools.org/?perspective=full&conceptId1=1251542004&edition=MAIN/2023-04-30&release=&languages=en))            | 29-2072 (Medical Records Specialists)                     |
+| Quality Assurance          | 56542007 ([Medical record administrator](https://browser.ihtsdotools.org/?perspective=full&conceptId1=56542007&edition=MAIN/2023-04-30&release=&languages=en)) | 15-1253 (Software Quality Assurance Analysts and Testers) |
+| Assistant                  | 449161006 ([Physician assistant](https://browser.ihtsdotools.org/?perspective=full&conceptId1=449161006&edition=MAIN/2023-03-31&release=&languages=en))        | 29-1071 (Physician Assistants)                            |
+
+Below is an example of a `Task.performerType` [CodeableConcept](/docs/fhir-basics#codeable-concepts-standarding-data) using both SNOMED and SOC systems.
+
+```ts
+{
+  resourceType: 'Task',
+  // ...
+  performerType: [
+    {
+      text:'Medical Billing Specialist',
+      coding:[
+        // Snomed
+        {
+          code:'1251542004',
+          system: 'http://snomed.info/sct',
+          display: 'Medical Coder'
+        },
+        // US SOC
+        {
+          code:"29-2072"
+          system: "https://www.bls.gov/soc"
+        }
+      ],
+    }
+  ]
+}
+```
+
+## Task focus
+
+The `Task.focus` element tracks the FHIR resource being _operated on_ by this task, known as the "focal resource". See the [Examples](#examples) section below for examples of focal resources in common scenarios.
+
+Well maintained `Task.focus` elements are critical data hygiene that streamlines operations and analytics . Making sure that every [`Task`](/docs/api/fhir/resources/task) has a populated`focus` reference will make it easier to find all touch points for a given clinical resource, spot operational bottlenecks and calculate turn-around-times, conversions, and care quality metrics as your implementation scales.
+
+## Task start / due dates
+
```

Comment:
Include common TAT queries - including GraphQL if needed.  CLIA/CAP has an example: https://www.medplum.com/docs/compliance/clia-cap

Reply:
I added a section `Task.executionPeriod`, with the turnaround time being `Task.executionPeriod.end` - `Task.executionPeriod.start`. 

However, there isn't a single query to compute the turnaround time itself since neither graphQL or standard search can perform that datetime math within the search.